crates/context/config/src/client/protocol/evm.rs

```diff
@@ -183,25 +182,20 @@ impl Network {
 
         let call_data = [method_selector, &args].concat();
 
-        let params = vec![
-            json!({
-                "to": address,
-                "data": Bytes::from(call_data)
-            }),
-            json!("latest"),
-        ];
+        let request = TransactionRequest::default()
+            .to(address)
+            .input(Bytes::from(call_data).into());
 
-        let response: Bytes = self
+        let bytes = self
             .provider
-            .client()
-            .request("eth_call", params)
+            .call(&request)
             .await
             .map_err(|e| EvmError::Custom {
                 operation: ErrorOperation::Query,
                 reason: format!("Failed to execute eth_call: {}", e),
             })?;
 
-        Ok(response.to_vec())
+        Ok(bytes.to_vec())
```

Comment:
[`Bytes: Into<Vec<u8>>`](https://docs.rs/alloy/latest/alloy/primitives/struct.Bytes.html#impl-From%3CBytes%3E-for-Vec%3Cu8%3E)

```suggestion
        Ok(bytes.into())
```

Reply:
Updated